applications/GeoMechanicsApplication/tests/test_load_on_block_2d_no_damping/README.md

```diff
@@ -0,0 +1,23 @@
+# Load on block 2D no damping
+
+This test verifies that in a block of plane strain elements, the motion induced by instant application of a line load does not damp out.
+
+## Setup
+
+This test consists of a 4 3-noded elements (using the UPwSmallStrainElement2D3N class) of $1x1 \mathrm{[m]}$. A
+schematic representation can be found in the figure below:
+![MeshStructure](MeshStructure.svg)
```

Comment:
I don't know why, but this one doesn't render for some reason: 
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/687ad21f-a1d6-41c1-b676-950433ccb41e)

Reply:
No clue, in CLion it shows.

Reply:
Just checked, if you add an extra blank line, I think it works

Reply:
added the line as suggested.